elixir/apps/web/lib/web/live/policies/new.ex

```diff
@@ -33,53 +36,64 @@ defmodule Web.Policies.New do
     <section class="bg-white dark:bg-gray-900">
       <div class="max-w-2xl px-4 py-8 mx-auto lg:py-16">
         <h2 class="mb-4 text-xl font-bold text-gray-900 dark:text-white">Policy details</h2>
-        <form action="#">
-          <div class="grid gap-4 mb-4 sm:grid-cols-1 sm:gap-6 sm:mb-6">
-            <div>
-              <.label for="policy-name">
-                Name
-              </.label>
-              <.input
-                autocomplete="off"
-                type="text"
-                name="name"
-                value=""
-                id="policy-name"
-                placeholder="Enter a name for this policy"
-              />
-            </div>
-            <div>
-              <.label for="group">
-                Group
-              </.label>
-
-              <.input
-                type="select"
-                options={Enum.map(@actor_groups, fn g -> [key: g.name, value: g.id] end)}
-                name="actor_group"
-                value=""
-              />
-            </div>
-            <div>
-              <.label for="resource">
-                Resource
-              </.label>
-              <.input
-                type="select"
-                options={Enum.map(@resources, fn r -> [key: r.name, value: r.id] end)}
-                name="resource"
-                value=""
-              />
-            </div>
-          </div>
-          <div class="flex items-center space-x-4">
-            <.button type="submit">
-              Save
+        <.simple_form
+          for={@form}
+          class="space-y-4 lg:space-y-6"
```

Comment:
I think this class should not have an effect because the `.simple_form` component already wraps everything in `<div class="space-y-8 bg-white">`. Maybe we should drop this component completely? Doesn't look like it has a lot of usage anyways.

Reply:
Yeah, I think you're right about the class not having any effect.

As far as getting rid of the component, I don't have a strong preference.  I started using the `.simple_form` component in the `sign_up.ex` page because I saw it was used in the `auth/sign_in.ex` page and then I used it here because I used it in the `sign_up.ex` page.

I think my only suggestion would be to leave it for now and then have another PR that goes through and removes it from everything.  That way this PR can stay focused to just the Policy New/Edit pages and then the new PR can focus on removing the component and all of it's uses.  Thoughts?